fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -2686,6 +2689,7 @@ public void replayDropDb(String dbName) throws DdlException {
             idToDb.remove(db.getId());
             final Cluster cluster = nameToCluster.get(db.getClusterName());
             cluster.removeDb(dbName, db.getId());
+            globalTransactionMgr.removeDatabaseTransactionMgr(db.getId());
```

Comment:
Same to `dropDb`, this should be called in `CatalogRecycleBin.replayEraseDatabase()`